app/utils/callback/lava.py

```diff
@@ -296,6 +296,7 @@ def store_lava_json(job_data, meta, base_path=utils.BASE_PATH):
     job_data["lab_name"] = meta.get("lab_name")
     job_data["version"] = meta.get("version")
     job_data["boot_log_html"] = meta.get("boot_log_html")
+    job_data["test_log_html"] = meta.get("log")
```

Comment:
`meta["log"]` is not HTML, it's a YAML log coming directly from LAVA.  So it should probably be called something else in `job_data`, such as `job_data["log"]` or `job_data["lava_log"]`.